editor/src/clj/editor/debugging/mobdebug.clj

```diff
@@ -326,7 +326,7 @@
   (try
     (doto (Socket.)
       (.connect (InetSocketAddress. ^String address (int port)) 2000))
-    (catch java.net.ConnectException _ nil)))
+    (catch java.lang.Exception _ nil)))
```

Comment:
We want to catch the SocketTimeoutException too

Reply:
Good! It makes sense to have very broad exception catching in this case.